src/community/design-system-day/speaker-information.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Speaker and panelist information for Design System Day 2023
+description: Speaker and panelist biographies and photographs 
+section: Community
+theme: Events and workshops
+layout: layout-pane.njk
+order: 13
+---
+
+<img src="/community/images/dsd23-announcement-banner.svg" alt="Design System Day 2023 logo" class="app-image--no-border" loading="lazy">
+
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "_image-card.njk" import imageCard %}
+
+
+{{ govukSummaryList({
+  classes: "govuk-!-margin-top-3 govuk-!-margin-bottom-9",
+  rows: [
+    {
+      key: {
+        text: "Dates"
+      },
+      value: {
+        text: "10 and 11 October 2023"
+      }
+    },
+    {
+      key: {
+        text: "Location"
+      },
+      value: {
+        html: '<a href="https://dynamicearth.org.uk/plan-your-visit/getting-here/">Dynamic Earth</a>, Edinburgh, and online'
+      }
+    }
+  ]
+}) }}
+
+## Speakers 
+
+Our speakers come from organisations across the UK and international public sector. We're joined by folks working on design systems from notable companies too.
+
+## Day 1 speakers
+
+### Kelly Lee (GOV.UK Design System team)
+
+Kelly Lee is a delivery manager on the GOV.UK Design System team. Before joining the Government Digital Service, she worked at Jobcentre Plus in a variety of roles including business coach, performance team leader and financial assessor.
```

Comment:
Everyone else only has first name.

```suggestion
Kelly is a delivery manager on the GOV.UK Design System team. Before joining the Government Digital Service, she worked at Jobcentre Plus in a variety of roles including business coach, performance team leader and financial assessor.
```

Reply:
I disagree, @calvin-lau-sig7. You only need to capitalise the job title when it's a proper noun, like on a business card or in an email signature. The rest of the time it's fine without capitals. 

`Kelly Lee is Delivery Manager on the GOV.‌UK Design System` would be OK but it also makes it sound like she's the only delivery manager.